spec.md

```diff
@@ -92,39 +91,51 @@ The `Span` interface must have the following capabilities:
                 +--------+--------+     CHILD OF SPAN C), AS WELL
                 |        |        |     AS SPANS G, H, AND I.
             [Span G] [Span H] [Span I]
-```
+~~~
 
 
-### Tracer
+## Tracer
 
 The `Tracer` interface must have the following capabilities:
 
-- Start a `Span` that has no parent, commonly referred to as a *root* `Span` **(py: `start_trace`, go: `StartTrace`)**
-- Provide some form of access to a SpanPropagator (whether that's through embedding, inheritance, containment, or something else is per-platform)
+- Start a new `Span`. There must be a way for the caller to specify a parent `Span`, an explicit start timestamp (other than "now"), and an initial set of `Span` tags. **(py: `start_span`, go: `StartSpanWithOptions`)**
+- Return an `Injector` instance given a format identifier (see [Injector and Extractor](#injector-and-extractor))
+- Return an `Extractor` instance given a format identifie (see [Injector and Extractor](#injector-and-extractor))
 
-### SpanPropagator
+`Injector` and `Extractor` both rely on "format identifiers." The nature of the "format identifier" may vary from platform to platform, but in all cases the format identifiers should come from a global namespace. New formats must not *require* changes to the core OpenTracing platform APIs, though those core platform APIs should define a few basic/general formats (like string maps, binary blobs, and/or HTTP headers). For example, if the maintainer of EsotericRPCFramework wanted to define an EsotericRPCFramework injection and extraction format, she or he must be able to do so without sending a PR to OpenTracing maintainers (though of course OpenTracing implementations are not required to suppor the EsotericRPCFramework format).
```

Comment:
I think I'll leave this here... it's really a comment about the "format identifiers" more than the Injectors/Extractors themselves and thus belongs in the Tracer section. I will reword, though.